daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/ReplayCommand.scala

```diff
@@ -20,9 +20,18 @@ object ReplayCommand {
       argument: Value,
   ) extends ReplayCommand
 
+  /** Exercise a template choice, by template Id or interface Id. */
+  final case class LenientExercise(
+      templateId: TypeConName,
+      contractId: Value.ContractId,
+      choiceId: ChoiceName,
+      argument: Value,
+  ) extends ReplayCommand
```

Comment:
I'm a bit confused. It looks like the implementation allows exercising only by template id?

Reply:
Maybe worth updating the comment.

Reply:
Good catch I will change the argument name,